### PR TITLE
[stable/postgresl] Point the chart to the postgresql release that includes postgis

### PR DESCRIPTION
**Pull request by rafariossaa**
Signed-off-by: Rafael Rios Saavedra <rafael.rios.saavedra@gmail.com>

#### What this PR does / why we need it:

Adds a note in the README about the inclusion of postgis in the postgresql container image.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Special notes for your reviewer:
This is a clean of the canceled https://github.com/helm/charts/pull/18354
Keep it on-hold until it get checked.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [X] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/postgresql/README.md [hunk 1]
```diff
@@ -371,6 +371,16 @@ $ helm upgrade my-release bitnami/influxdb \
 
 > Note: you need to substitute the placeholders _[POSTGRESQL_PASSWORD]_, and _[REPLICATION_PASSWORD]_ with the values obtained from instructions in the installation notes.
 
+## 6.6.0
+
+In this version, the chart will use PostgreSQL with the Postgis extension included.  The version used with Postgresql version 10, 11 and 12 is Postgis 2.5. It has been compiled with the following dependencies:
```
Comment (carrodher) [inline stable/postgresql/README.md]:
```suggestion
In this version, the chart will use PostgreSQL with the Postgis extension included. The version used with Postgresql version 10, 11 and 12 is Postgis 2.5. It has been compiled with the following dependencies:
```

**Comment by juan131** [comment]
Please bump the version @rafariossaa

**Pull request merged**